steps/confirmwinpeplainisodoesnotexistonmacosorlinuxworker/script.txt

```diff
@@ -0,0 +1,40 @@
+if [[ {kickstartedBootLoaderIsBios} == {kickstartedBootLoaderIsUefi} ]]
```

Comment:
This check is a bit off.
If you want to go all out, the checks should be

First, check that the values are either "true" or "false" strings, then check the values

if # check that the value strings are true or false

elif {kickstartedBootLoaderIsBios} && {kickstartedBootLoaderIsUefi}

elif {kickstartedBootLoaderIsBios}

elif {kickstartedBootLoaderIsUefi}

else
ERROR
fi


But rather than fix it, You should delete the step.
If the file already eixsts, delete it in Attune. Why get someone to delete it manually when we're going to rebuild it.